hail/python/hailtop/utils/utils.py

```diff
@@ -75,7 +75,7 @@ def first_extant_file(*files: Optional[str]) -> Optional[str]:
     return None
 
 
-def cost_str(cost):
+def cost_str(cost: Optional[int]) -> Optional[int]:
```

Comment:
What should the return type be? Hint: look at the name of the function and what is being returned on line 81.